qiskit/algorithms/optimizers/adam_amsgrad.py

```diff
@@ -246,6 +246,9 @@ def optimize(
                 point: is a 1D numpy.ndarray[float] containing the solution\n
                 value: is a float with the objective function value\n
                 nfev: is the number of objective function calls
+
+         Note: You should add seed via algorithm_global 
```

Comment:
Can we change this text - saying you 'should' seed it implies you ought to do this - in general its random since we want it normally to be random. If you want to reproduce behavior, then you would want to seed the random generator. Something more like `Note: This component has some function that is normally random. If you want to reproduce behavior then you should set the random number generator seed in the algorithm_globals`. Also as was mentioned the text would be better off in the class docstring itself.

It would also be possible, rather than saying "...has some function...", as I did in the text above, which does not state explicitly what it is, to make that more explicit for each component so that the reader has a better idea of what the aspect is that is random. E.g if in VQE you explicitly set the initial_point then seeding, which generates a random initial_point if none is provided, will make no difference. It does not harm of course to set the seed, but that will not change the behavior of the VQE instance in this case since it never invokes the random logic,

Reply:
I will make the changes.